src/olympia/access/tests.py

```diff
@@ -71,8 +71,9 @@ class ACLTestCase(TestCase):
 
     def test_admin_login_anon(self):
         # Login form for anonymous user on the admin page.
-        url = '/en-US/admin/'
-        self.assertLoginRedirects(self.client.get(url), to=url)
+        url = '/en-US/admin/models/'
+        self.assert3xx(self.client.get(url),
+                       '/admin/models/login/?next=/en-US/admin/models/')
```

Comment:
The changes in `urls.py` causes an intermediary redirect to happen. We redirect to `/admin/models/login/?next=/en-US/admin/models/` which in turn redirects to FxA.

Reply:
Please check the changes in urls.py, maybe I made a mistake there, or there is a better way to do that.

Reply:
the urls are fine, it's just the redirections... does `follow=True` in `.get(url, follow=True)` help `assertLoginRedirects` work?  For the ACL tests especially I think testing it does actually end up in the login flow is ideal.

Reply:
That doesn't seem to work either. https://gist.github.com/wagnerand/8931cd78cfe02c887d9c82978ee5e7fb

Reply:
The problem is, we have extra redirects because of the way the django admin does it.
`self.client.get('/en-US/admin/models/')` redirects to `/admin/models/login/?next=/en-US/admin/models/` which redirects to `/en-US/admin/models/login/?next=/en-US/admin/models/` which redirects to FxA.

You can't use `follow=True` because it would follow until the end and try to load FxA url, which isn't allowed in tests as it's external.